index.html

```diff
@@ -1,10 +1,40 @@
+<!DOCTYPE html>
+
 <html>
   <head>
+    <title>What's For Dinner?</title>
     <link rel="stylesheet" href="./styles.css">
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
   <body>
-    
+    <header>
+      <h1 class="title">What's For Dinner?</h1>
+    <header>
+    <section class="white-box">
+      <div class="box-contents">
+        <h2>What are you looking for?</h1>
+        <div class="radio-btn">
+          <input type="radio" name="dish" value="sides">
+          <label class="dish-type"for="sides">Side</label>
+        </div>
+        <div class="radio-btn">
+          <input  type="radio" name="dish" value="main-dishes">
+          <label class="dish-type"for="main-dishes">Main Dish</label>
+        </div>
+        <div class="radio-btn">
+          <input type="radio" name="dish" value="desserts">
+          <label class="dish-type" for="desserts">Dessert</label>
+        </div>
+        <button class="lets-cook-btn">LET'S COOK!</button>
+      </div>
+    </section>
+    <section class="white-box">
+      <svg class="pot-img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 125">
+          <path d="M84.883 42.8h-1.346v-4.504a1 1 0 00-1-1H17.713a1 1 0 00-1 1V42.8h-1.346c-2.891 0-5.242 2.352-5.242 5.242s2.352 5.242 5.242 5.242h1.346v32.874a4.003 4.003 0 003.998 3.998h58.828a4.003 4.003 0 003.998-3.998V53.284h1.346c2.891 0 5.242-2.352 5.242-5.242S87.773 42.8 84.883 42.8zm-21.594-3.504h5.172v10.845c0 1.426-1.16 2.585-2.586 2.585s-2.586-1.159-2.586-2.585V39.296zm-12.403 0h5.171v16.618c0 1.426-1.16 2.586-2.586 2.586s-2.585-1.16-2.585-2.586V39.296zM15.367 51.284c-1.788 0-3.242-1.454-3.242-3.242s1.454-3.242 3.242-3.242h1.346v6.484h-1.346zm66.17 34.874a2 2 0 01-1.998 1.998H20.711a2 2 0 01-1.998-1.998V39.296h30.173v16.618c0 2.528 2.057 4.586 4.585 4.586s4.586-2.058 4.586-4.586V39.296h3.232v10.845c0 2.528 2.058 4.585 4.586 4.585s4.586-2.057 4.586-4.585V39.296h11.076v46.862zm3.346-34.874h-1.346V44.8h1.346c1.788 0 3.242 1.454 3.242 3.242s-1.454 3.242-3.242 3.242zM19.33 22.125a5.55 5.55 0 00-4.898 6.121l.426 3.842a1 1 0 001.104.884l67.588-7.498a1 1 0 00.884-1.104l-.427-3.842a5.506 5.506 0 00-2.046-3.717 5.52 5.52 0 00-4.075-1.182L54.799 18.19l-.229-2.06.406-.045a1 1 0 00.884-1.104l-.454-4.09a.999.999 0 00-1.104-.884l-14.064 1.561a1 1 0 00-.884 1.104l.454 4.09a1 1 0 001.104.884l.442-.049.228 2.06-22.252 2.468zm58.776-4.508a3.517 3.517 0 012.604.755 3.522 3.522 0 011.308 2.376l.316 2.848-65.6 7.277-.315-2.848a3.546 3.546 0 013.131-3.912l58.556-6.496zm-24.578-5.512l.233 2.102-12.076 1.34-.233-2.102 12.076-1.34zm-10.185 5.271l9.239-1.025.229 2.06-9.239 1.025-.229-2.06z"/>
+      </svg>
+      <p class="hidden">You should make:<br><span class="random-dish"></span></p>
```

Comment:
Is there a reason you need the span inside the p tag here? (p tag normally means an un-broken paragraph rather than header+subheader like how this renders) You could apply hidden class to both elements if you want to avoid that. Just remember that p and span tags have different default styles like the display property.

Reply:
I used it so I could style the sentence in two different ways. "You should make" has different styling than the random text I insert inside the span. I wasn't sure how to do that so I used span to represent another class for styling. Do you have a suggestion of how I could set this up better in HTML? Should I have just used two h tags instead? @patrickwhardy

Reply:
Gotchya. Yes, I would have two different classes like you do. The only funky part is that the span is nested _inside_ the p tag. You could do something like this:
```
<div class="hidden">
  <p class="dish-header">You should make:</p>
  <p class="random-dish"></p>
</div>
```

Reply:
Ok, thanks! I think in my head it was one sentence that I needed to break up and wasn't sure about labeling two parts of a sentence as a paragraph with the p tag. I don't think I understand the <span> tag. What is an example of proper use for this?

Reply:
You're right that ideally a p tag wraps a paragraph. In practice I use p or spans based on their styles: p tags are display block and spans are inline. So it's really just a style preference for me that when I see two p tags on two different lines, I know it's probably going to render two lines of text. I would use your approach if I wanted to do something like bold a word in a sentence e.g.
```
<p>The <span class="bold">fox</span> jumps over the log</p>
```

Reply:
Ok I understand. I had my paragraph broken up on two lines and it makes more sense to just use two block text elements and apply the styling that way. Thanks!